src/main/java/io/papermc/lib/environments/PaperEnvironment.java

```diff
@@ -38,6 +40,9 @@ public PaperEnvironment() {
                 bedSpawnLocationHandler = new BedSpawnLocationPaper();
             } catch (NoSuchMethodException ignored) {}
         }
+        if (isVersion(16, 4)) {
+            getOfflinePlayerHandler = new GetOfflinePlayerCachedOption();
```

Comment:
needs `Server.class.getDeclaredMethod("getOfflinePlayerIfCached", String.class)` check before assigning this